rts/lra/trip-controller/src/main/java/io/narayana/rts/lra/demo/tripcontroller/TripCheck.java

```diff
@@ -37,14 +39,29 @@ static boolean validateBooking(Booking booking, boolean isConfirm, WebTarget hot
         });
 
         if (isConfirm) {
-            // the hotel and only one of the flight bookings should be confirmed
-            if (confirmCount[0] != 2 || cancelCount[0] != 1) {
-                System.out.printf("TripCheck: validateBooking: the hotel and only one of the flight bookings should have been confirmed%n");
+            if (confirmCount[0] != 3) {
+                try {
+                    System.out.println(
+                            "TripCheck: validateBooking: the hotel and 2 flight bookings should have been confirmed, but are "
+                                    + confirmCount[0] + "\n" + booking.toJson());
+                }
+                catch (JsonProcessingException e) {
+                    // TODO Auto-generated catch block
```

Comment:
I would suggest to remove the TODO here and in the next block you added but again not blocking here though

Please can I ask you to confirm that JsonProcessingException is expected to be thrown though? If it's not expected to be thrown I think you should back out that handling. I will not block on that either.

Reply:
I'll delete the comment, that exception can be thrown by booking.toJson(), see https://github.com/jbosstm/quickstart/blob/main/rts/lra/trip-client/src/main/java/model/Booking.java#L149-L153